### DISCUSSION TITLE
I want to put a resource pack in Farmers Delight mod on the server. Will it be possible?

### DISCUSSION THREAD

**Discussion by LaeDev** [vectorwing/FarmersDelight#580] [Q&A]
Hello. I want to use pictures of food in Farmers Delight and other block models for vanilla server. I know the MIT license is applied, but just in case, I'm asking the author one more time :) Is this request possible?

**Comment by vectorwing** [comment]
Sure! I only ask that you make it clear somewhere to your users that I'm the author of these textures. Feel free to draw derivative sprites using elements from FD's assets as well. 👍

**Reply by LaeDev** [reply]
Thank you for your permission. I'm going to write down the resource of FD in the server resource pack file. Would you like to write it down somewhere else?
